client/blocks/subscription-length-picker/test/index.js

```diff
@@ -0,0 +1,100 @@
+/** @format */
+
+jest.mock( 'lib/abtest', () => ( {
+	abtest: () => '',
+} ) );
+
+const translate = x => x;
+jest.mock( 'i18n-calypso', () => ( {
+	localize: ComposedComponent => {
+		const React = require( 'react' );
+		const componentName = ComposedComponent.displayName || ComposedComponent.name || '';
+		return class Wrapper extends React.Component {
+			static displayName = componentName;
+
+			render() {
+				return (
+					<ComposedComponent
+						{ ...this.props }
+						translate={ function( x ) {
+							return x;
+						} }
+					/>
+				);
+			}
+		};
+	},
+	numberFormat: x => x,
+} ) );
+
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+import { shallow } from 'enzyme';
+import React from 'react';
+import { getPlan } from 'lib/plans';
+import { PLAN_BUSINESS, PLAN_BUSINESS_2_YEARS } from 'lib/plans/constants';
+
+/**
+ * Internal dependencies
+ */
+import { SubscriptionLengthPicker } from '../index';
+
+describe( 'SubscriptionLengthPicker basic tests', () => {
```

Comment:
I personally don't find much value in component tests that are not base components. Unit tests for new selectors/data functions tend to be more valuable. Use your best judgement here and see if you'd like to keep these.

Reply:
I do not believe in 100% test coverage, but I believe in coverage that's good enough and protects us from making mistakes. These tests are ensuring the component can be rendered and that it fulfils it's general task and so I am a proponent of having them.